utils/vue.js

```diff
@@ -0,0 +1,54 @@
+import Vue from 'vue';
+
+const ellxify = Component => class {
+  constructor(props, { initState }) {
+    this.value = initState;
+    this.target = document.createElement('div');
+    this.emit = () => {};
+
+    const ComponentClass = Vue.extend(Component);
+
+    this.instance = new ComponentClass({
+      propsData: {
+        value: this.value,
+        ...props
+      }
+    });
+
+    this.instance.$watch('value', (v) => this.emit(v));
+  }
+
+  stale() {
+    this.instance.$props.stale = true;
+  }
+
+  update(props = {}) {
+    for (let key in props) {
+      if (key === '__EllxMeta__') continue;
```

Comment:
why special treatment ?

Reply:
should we ever be passing it to components?

Reply:
There's no obvious harm. If we ever encounter a situation where this is undesirable I would prefer just to make it non-enumerable in ellxify.

Reply:
removed